[line 4]: Error at 'foo': Object is not a callable.